domains/_vercel.daim.json

```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "devdaim6",
+    "email": "daimdev6@gmail.com"
+  },
+  "records": {
+    "TXT": [
+      "vc-domain-verify=daim.is-a.dev,d88985a4876213470d49",
```

Comment:
Do the subdomains for these domains exist?

Reply:
![image](https://github.com/user-attachments/assets/932a7719-e057-455b-94aa-69de93595427)